alt_src/alt_src.py

```diff
@@ -757,6 +757,16 @@ def sync_repo(self):
         cmd = ['git', 'fetch', '-v', git_url, '+refs/*:refs/*']
         try:
             self.log_cmd(cmd, cwd=repo)
+        except CommandError:
+            self.logger.warning('Unable to fetch remote repo')
+            self.logger.warning('Local cache exists: %s', repo)
+            self.logger.warning('Trying to reclone repo')
+            # remove broken/empty dir
+            self.logger.warning('Removing empty or broken repo')
+            shutil.rmtree(repo)
+            # reclone repo
+            self.logger.warning('Clonning new repo')
```

Comment:
```suggestion
            self.logger.warning('Cloning new repo')
```

... or maybe this will be fixed by the logging changes Robert was asking for anyway.